neutrino.go

```diff
@@ -783,6 +784,13 @@ func NewChainService(cfg Config) (*ChainService, error) {
 		})
 	}
 
+	s.utxoScanner = NewUtxoScanner(&UtxoScannerConfig{
+		BestSnapshot:       s.BestSnapshot,
+		GetBlockHash:       s.GetBlockHash,
+		BlockFilterMatches: s.blockFilterMatches,
```

Comment:
Should this be a follow up once batching is finished? Or is there something you have in mind that's currently actionable?

Reply:
Yeh IMO should be a follow up rather than modifying this PR to implement. When we implement batch utxo fetching for the routine "catch up to tip" behavior, then we can modify this at that time as well.